engines/group/app/serializers/group/base_serializer.rb

```diff
@@ -0,0 +1,5 @@
+module Group
+  class BaseSerializer < Oat::Serializer
+    adapter FlatAPI
```

Comment:
What I see doing a rails in this way (without `app` folder) is that we are generating in every engine this kind of base serializer file instead of inherit from a global base serializer